File: VendingMachinesSys/VendingMachinesSysAPI/Models/VendingMachine.cs
using System.ComponentModel.DataAnnotations;

namespace VendingMachinesSysAPI.Models
{
    public class VendingMachine
    {
        [Key]
        public int SerialNumber { get; set; }

        [Required]
        public string Locations { get; set; }

        [Required]
        public double Latitude { get; set; }

        [Required]
        public double Longitude { get; set; }

    }
}

File: VendingMachinesSys/VendingMachinesSysWebApp/ClientApp/src/domain/VendingMachine.ts
export class VendingMachine {
  serialNumber: number;
  locations: string;
  latitude: number;
  longitude: number;

  constructor(serialNumber: number, locations: string, latitude: number, longitude: number) {
    this.serialNumber = serialNumber;
    this.locations = locations;
    this.latitude = latitude;
    this.longitude = longitude;
  }
}
